Automattic/jetpack
# Analyzer: Update to Slurper script

- Increase memory when running from command line
- Update example script
- Update permissions so Slurper is executable




In running the Slurper script, I made a few modifications to help it run better, so paying it forward. Does not impact the actual anaylizer.

#### Changes proposed in this Pull Request:

* See above.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* You can run the whole slurper if you want to, but the key is to have a directory that has a few directories of plugins. Use that for the $slurper_path.
* Point the new path to a new built version of JP (downloading from jetpack-production or the stable version from w.org works.)
* Point the old path to a recent version from w.org. The example assumes you have SVN jetpack checked out somewhere.
* Run `projects/packages/analyzer/scripts/jetpack-slurper.php` as such from the command line (do not prepend with `php`)

It should execute. Before this PR, you'd need to run `php [file path]`. The big savings here is the shebang line sets the memory to 2 GB.
